src/rhoknp/cli/serve.py

```diff
@@ -0,0 +1,150 @@
+import html
+import textwrap
+from enum import Enum
+from typing import Dict, Optional, Union
+
+try:
+    import fastapi
+    import uvicorn
+except ImportError:
+    fastapi = None
+    uvicorn = None
+
+from rhoknp.processors import KNP, KWJA, Jumanpp
+
+
+class AnalyzerType(Enum):
+    """解析器の種類．"""
+
+    jumanpp = "jumanpp"
+    knp = "knp"
+    kwja = "kwja"
+
+
+BASE = textwrap.dedent(
+    """\
+    <!DOCTYPE html>
+    <html lang="ja">
+    <head>
+        <meta charset="UTF-8">
+        <title>rhoknp</title>
```

Comment:
Showing the analyzer type in the title is more informative rather than `rhoknp`.